### ISSUE TITLE
No mapping for the Unicode character exists in the target multi-byte code page.

### ISSUE THREAD

**Issue by aajakey** [rileytestut/AltStore#1056]
When trying to install AltStore on my phone from Windows 11, I get this error message.

<img width="322" alt="Screenshot 2022-09-15 093120" src="https://user-images.githubusercontent.com/67519074/190342780-4c79e7ed-bdbc-43a2-98a9-491bfa68aac7.png">

**Comment by aajakey** [comment]
I think I figured out why this bug exists. My user folder, i.e. "C:\Users\USERNAME" contains a special character (ć). 

Creating a new account on the same PC without those characters is a workaround until this issue gets resolved. 

Also, this doesn't affect refreshing apps, only installing AltStore for the first time.

**Comment by lonkelle** [comment]
Good catch, we should fix this. Thanks for reporting both it and your fix!

**Comment by frontagedev** [comment]
Thanks
